### PR TITLE
Updated links on Build an Accessible Extension Page

### LINKED ISSUE #362: Update links on Build an Accessible Extension Page
Related to #245. 

All of the links that are 404'ing in the `build-an-accessible-extension.md` file need updating. 

There is a good chance that replacing the prefix`https://https://extensionworkshop-dev.allizom.org` with `https://developer.mozilla.org` will lead you to the MDN page for that topic. We need these links to point to those MDN pages, without the localization coding.

### PR DESCRIPTION
**Pull request by ruchikabgosain**
Fixes #362 

All of the links that were 404'ing in the `build-an-accessible-extension.md` file were updated and localization coding was removed. 

cc: @caitmuenster

### CODE REVIEW

**Comment by caitmuenster** [comment]
@ruchikabgosain I'm EOD'ing but this is on my list for review tomorrow.

**Comment by ruchikabgosain** [comment]
Sure @caitmuenster, my best to you 😄

File: documentation/develop/build-an-accessible-extension.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ There is more information on design and accessibility in the [Photon Design Syst
 <tr>
 <td markdown="1">
 
-[Keyboard shortcuts (commands)](/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/commands)
+[Keyboard shortcuts (commands)](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/commands)
```
Comment (championshuttler) [inline documentation/develop/build-an-accessible-extension.md]: All the links should not have `/en-US` which is the locale part in urls. For example here URL should be `https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/commands`

File: documentation/develop/build-an-accessible-extension.md [hunk 2]
```diff
@@ -67,12 +67,12 @@ To improve accessibility, add keyboard shortcuts for:
 <tr>
 <td markdown="1">
 
-[Toolbar button (browser action)](/Add-ons/WebExtensions/user_interface/Browser_action)
+[Toolbar button (browser action)](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/user_interface/Browser_action)
```
Comment (championshuttler) [inline documentation/develop/build-an-accessible-extension.md]: Same here.

File: documentation/develop/build-an-accessible-extension.md (hunk 1 above)
Comment (ruchikabgosain) [reply documentation/develop/build-an-accessible-extension.md]: okay!

**Comment by ruchikabgosain** [comment]
Did the required changes @championshuttler @caitmuenster , let me know if this works 😺

**Pull request merged**